CHANGELOG.asciidoc

```diff
@@ -3,6 +3,574 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.8.0]]
+=== Beats version 7.8.0
+https://github.com/elastic/beats/compare/v7.7.0...v7.8.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Introduce APM libbeat instrumentation, active when running the beat with ELASTIC_APM_ACTIVE=true. {pull}17938[17938]
```

Comment:
```suggestion
- Introduce APM instrumentation, which is active when running the beat with `ELASTIC_APM_ACTIVE=true`. {pull}17938[17938]
```

Reply:
Can we drop "libbeat" from this description? Just thinking that libbeat is an implementation detail that end users don't really need to know about.